ADD: fade in and out on flash messages.

JRRS1982 opened the pull request:
Just making the flash messages a bit prettier. Have added in/delay/out onto the javascript
to make things just a little bit nicer. Happy to change timings, just let me know.

JRRS1982 committed:
ADD: fade in and out on flash messages.

Just making the flash messages a bit prettier. Have added in/delay/out onto the javascript
to make things just a little bit nicer.

JRRS1982 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Make flash messages fade out rather than disappear](https://trello.com/c/aQpXaMvR/40-make-flash-messages-fade-out-rather-than-disappear)

arthurashman commented:
Got no issue with this. But might want to see it in action to see if the fade or timing want modifying. Otherwise happy to approve merge.

JRRS1982 committed:
EDIT: doubling time the message is displayed for. 

As per team requests, have doubled the display time as may need longer to read the message.
